.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,23 @@
 <!--Thanks for your contribution. See [CONTRIBUTING](CONTRIBUTING.md)
-    for this project's contribution guidelines. -->
+    for this project's contribution guidelines. Remove these comments
+    as you go.
 
-<!--DO NOT edit files and directories listed in .NOT_EDITED_HERE.yaml.
-    These are maintained in upstream repos and changes here will be lost.-->
+    DO NOT edit files and directories listed in .NOT_EDITED_HERE.yaml.
+    These are maintained in upstream repos and changes here will be lost.
+
+    Details are appreciated and labels / milestones / reviewers are helpful -->
 
 ### Describe the proposed changes
```

Comment:
How about just "Proposed Changes" so we have to edit less, and it's the same format as the other headings?